app/src/main/res/values/strings.xml

```diff
@@ -422,6 +422,8 @@ and always knows where you are to not miss where to get off the bus.
 	<string name="np_desc_cmta">Austin</string>
 	<string name="np_name_usny">New York and New-Jersey States</string>
 	<string name="np_desc_usny" translatable="false">MTA, CDTA, Port Authority, Staten Island Ferry, NFTA, Rockland County, Rochester-Genesee Regional Transportation Authority</string>
+	<string name="np_name_usdc">Washington, DC</string>
+	<string name="np_desc_usdc" translatable="false">Metro</string>
```

Comment:
Is it really only metro?

Reply:
And Metrobus and Capital Bikeshare (which I just added to the string). I also sent a message to Navitia asking if they could add ART bus data.

Is there a way I could use the Navitia playground to see all the services provided? Right now I'm just sending requests and looking at the results. I can't tell if I have everything.

Reply:
Hi @liamnorm, what is your opinion regarding WMATA? Is it worth including in the network description string?

Reply:
Well, Metro is the same thing is WMATA (the 'M' is for Metropolitan). Everyone I know calls it Metro. I don't think adding WMATA is necessary.